build/lib/run_target_docker.sh

```diff
@@ -92,11 +92,16 @@ if [[ "$SKIP_RUN" == "false" ]]; then
 		DOCKER_RUN_NETRC=""
 	fi
 
+	PRIVILEGED=""
+	if [[ "$REMOVE" == "false" ]]; then
+		PRIVILEGED="--privileged --mount type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock"
```

Comment:
i dont think we need to run as privileged, i have tried building on al2 and mac and it seems fine to not include this flag.  only including it for the case that someone uses start-builder target to start the long lived container